india_compliance/gst_india/utils/gstr_2/gstr_2b.py

```diff
@@ -19,29 +19,22 @@ def get_supplier_details(self, supplier):
             "sup_return_period": supplier.supprd,
         }
 
-    def get_transaction_item(self, item):
```

Comment:
check for backwards compatibility. This should work even if downloading for old data.